packages/webservice/app/src/main/resources/db/changelog/changelog.xml

```diff
@@ -108,4 +123,9 @@
         );
         </sql>
     </changeSet>
+    <changeSet author="admin" id="3" runAlways="true" context="test">
+        <sql>
+          DELETE FROM PUBLIC.DATABASECHANGELOG where id = 2;
```

Comment:
Would it work if we moved this to step 1 and drop the entire changelog?

Reply:
It does not work, because it retrieves the changelog and determines what `changeset id` to run at the beginning.